WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -72,16 +74,21 @@ private void StepInputRegistrationPhase()
 				if (round.InputCount < Config.MinInputCountByRound)
 				{
 					Rounds.Remove(round.Id);
+					round.LogInfo($"Round {round.Id} removed because the number of inputs was not enough.");
```

Comment:
This will duplicate round.id in the output, I think.

Reply:
Correct. If you use round.LogInfo will automatically add the Round number for you.

Reply:
Done